src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -968,6 +968,21 @@ protected function addObjectAccessorBody(&$script, Column $column)
         return \$this->$cloUnserialized;";
     }
 
+    protected function addJsonAccessor(&$script, Column $column)
+    {
+        $this->addDefaultAccessorComment($script, $column);
+        $this->addDefaultAccessorOpen($script, $column);
+        $this->addJsonAccessorBody($script, $column);
+        $this->addDefaultAccessorClose($script);
+    }
+
+    protected function addJsonAccessorBody(&$script, Column $column)
+    {
+        $clo = $column->getLowercasedName();
+        $script .= "
+        return json_decode(\$this->$clo);";
```

Comment:
Well, $this->$clo is null and you call thus `json_decode(null);`. I don't see a json_encode in that line.